src/components/layout/header/UserMenu/UserMenu.module.scss

```
@@ -0,0 +1,57 @@
+.container {
+  @include column-flexbox($gap: 1.6rem);
+
+  width: 23rem;
+  padding: 1.6rem;
+
+  background: $opacity-white-10;
+  backdrop-filter: blur(1.6rem);
+  border: 0.1rem solid $opacity-white-20;
+  border-radius: 1.6rem;
+  box-shadow: $popup-shadow;
+}
+
+.container-user {
+  @include column-flexbox($gap: 0.8rem);
+}
+
+.container-user-info {
+  @include column-flexbox($gap: 0.2rem);
+}
+
+.container-nickname {
+  @include flexbox($gap: 0.2rem);
+}
```

상단에 .container 가 있어서, 그 안에 중첩 구문으로 다 넣어줘도 좋을것 같습니다!

```suggestion
.container {
  @include column-flexbox($gap: 1.6rem);

  width: 23rem;
  padding: 1.6rem;

  background: $opacity-white-10;
  backdrop-filter: blur(1.6rem);
  border: 0.1rem solid $opacity-white-20;
  border-radius: 1.6rem;
  box-shadow: $popup-shadow;

  &-user {
    @include column-flexbox($gap: 0.8rem);
  }

  &-user-info {
    @include column-flexbox($gap: 0.2rem);
  }

  &-nickname {
    @include flexbox($gap: 0.2rem);
  }
 }
```

---

반영했습니다!